tests/phpunit/api/v3/MembershipStatusTest.php

```diff
@@ -136,6 +136,26 @@ public function testDelete() {
     $result = $this->callAPISuccess('membership_status', 'delete', $params);
   }
 
+  /**
+   * Test that after checking the person as 'Deceased', the Membership is also 'Deceased' both through inline and normal edit.
+   */
+  public function testDeceasedMembershipInline() {
+    $membershipStatusID = $this->membershipStatusCreate();
```

Comment:
Actually the class you are  using  already created  a membershipStatus  in the setup  so you  can use  that - from the setup

```
    $this->_contactID = $this->individualCreate();
    $this->_membershipTypeID = $this->membershipTypeCreate(['member_of_contact_id' => $this->_contactID]);
    $this->_membershipStatusID = $this->membershipStatusCreate('test status');
```

Reply:
If I use, the membershipStatus in the setup, i think the teardown function will automatically delete that membership so then I won't have to call the delete function and it will pass the 2nd test as well. Am I right? or will I have to call the delete function even then?